wpeverest/everest-forms
# Enhancement - Delete all logs at a time.

### All Submissions:




### Changes proposed in this Pull Request:

Added 'Delete all logs' button for deleting all logs.

### How to test the changes in this Pull Request:

1. Go to Everest Forms > Tools > Logs.
2. Verify, 'Delete all logs' button working properly or not.

### Types of changes:




### Other information:




### Changelog entry

> Enhancement - Delete all logs at a time.
